app/models/forms/bulk_upload_lettings/prepare_your_file.rb

```diff
@@ -10,9 +10,7 @@ class PrepareYourFile
 
       def view_path
         case year
-        when 2022
-          "bulk_upload_lettings_logs/forms/prepare_your_file_2022"
-        else
+        when 2023
```

Comment:
This and a couple of other case statements were defaulting to 2023 previously, but they wouldn't anymore, not sure if there'd be a situation where it would need to.
Could change it to always return 2023 without the case statement, but thought I'd keep the case here so that it's clear where to add 2024 stuff. Not sure if that's the better approach tho

Reply:
I think this is fine, might be worth raising in PO review  - I know we're not doing e.g. legacy templates in 24/25 so that's somewhere it definitely doesn't make sense to have a default value, so I think this approach is better